Source/Eto/Forms/Taskbar.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace Eto.Forms
+{
+	/// <summary>
+	/// Methods to interact with application taskbar button.
+	/// </summary>
+	[Handler(typeof(IHandler))]
+	public static class Taskbar
+	{
+		/// <summary>
+		/// Sets the state and progress of the application.
+		/// </summary>
+		/// <param name="state">Taskbar button state.</param>
+		/// <param name="progress">Progress in range from 0.0f to 1.0f.</param>
+		public static void SetProgress(TaskbarProgressState state, float progress)
+		{
+			if (progress < 0.0f || progress > 1.0f)
+				throw new ArgumentOutOfRangeException(nameof(progress), "Progress needs to be in 0.0f to 1.0f range.");
+
+			var handler = Platform.Instance.Create<IHandler>();
```

Comment:
this should use `Platform.Instance.CreateShared<IHandler>()` so it only creates a single instance of the handler.